app/services/chat/message_converter.py

```diff
@@ -1,9 +1,11 @@
 # app/services/chat/message_converter.py
 
 from abc import ABC, abstractmethod
+import re
 from typing import Any, Dict, List, Optional
 
 SUPPORTED_ROLES = ["user", "model", "system"]
+IMAGE_URL_PATTERN = r'(https://img\.picgo\.net/[^\s)\]\"\']+)'
```

Comment:
image url设置死了，换成其他的图床就失效了

```suggestion
r'\[image\]\((.*?)\)'
```

Reply:
已修改